test/test_helper.rb

```diff
@@ -44,7 +44,7 @@ def sqlserver?
       self.class.sqlserver?
     end
 
-    def close_client(client=@client)
+    def close_client(client=(@client ||= nil))
```

Comment:
This seems really odd. Is this doing an assignment to the `@client` instance variable? When when the method is called, I assume? What's the scenario where this is happening and we want `@client` to be nil?

Reply:
I agree it's odd. It's strange that we're even calling this method without a param in the first place when `@client` has not been initialized. This was an attempt to get rid of `warning: instance variable @client not initialized` warnings while running tests locally. I don't feel strongly about keeping it since we probably should just fix the places we're calling this from.

Reply:
Would this do it?

```ruby
def close_client(client=nil)
  client ||= @client
  (...existing code...)
end
```

Reply:
Yeah I think so. I'll give it a try.

Reply:
Well, now I'm really confused because that change causes some tests to fail and I have no idea why 🤷

Reply:
Another option would be to revert all this just for this PR, and put in a future PR to fix the warning